biojava-integrationtest/src/test/java/org/biojava/nbio/structure/test/xtal/TestCrystalBuilder.java

```diff
@@ -104,6 +104,30 @@ public void test4MF8() throws IOException, StructureException {
 
 	}
 
+	@Test
+	public void test1AUY() throws IOException, StructureException {
+		// a virus with NCS operators
+		AtomCache cache = new AtomCache();
+		StructureIO.setAtomCache(cache);
+		cache.setUseMmCif(true);
+		Structure s1 = StructureIO.getStructure("1AUY");
+		CrystalBuilder cb = new CrystalBuilder(s1);
+		StructureInterfaceList interfaces = cb.getUniqueInterfaces(5.5);
+		assertTrue(interfaces.size()==12);
```

Comment:
Could you make this an assertEquals? Minor thing, but it makes the test reports nicer (on failures). Same below.